src/DSharp.Script/test/clearKeys.spec.js

```diff
@@ -0,0 +1,11 @@
+
+describe('clearKeys', () => {
+    test('missing key returns false', () => {
+        const clearKeys = require('../dist/ss.js').clearKeys;
```

Comment:
Is there a way we can move `'../dist/ss.js'` to a const somewhere for all specs to use?